### PR TITLE
Create first conda-forge 2.2.2 fine recipe

### PR DESCRIPTION
**Pull request by julian-belina**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by julian-belina** [comment]
I accept to be maintainer of this package

**Comment by julian-belina** [comment]
@conda-forge/staged-recipes , please review and merge my request

File: recipes/fine/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+package:
+  name: fine
+  version: 2.2.2
```
Comment (ocefpaf) [inline recipes/fine/meta.yaml:3]:
```suggestion
  version: {{ version }}
```

File: recipes/fine/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
+package:
```
Comment (ocefpaf) [inline recipes/fine/meta.yaml:1]:
```suggestion
{% set version = "2.2.2" %}

package:
```

File: recipes/fine/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  name: fine
+  version: 2.2.2
+
+source:
+  url: https://github.com/FZJ-IEK3-VSA/FINE/archive/refs/tags/v2.2.2.tar.gz
```
Comment (ocefpaf) [inline recipes/fine/meta.yaml:6]:
```suggestion
  url: https://github.com/FZJ-IEK3-VSA/FINE/archive/refs/tags/v{{ version }}.tar.gz
```

File: recipes/fine/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  source_files:
+    - .
+  requires:
+    - pytest
+  commands:
+    - pytest test/
+
+about:
+  home: https://github.com/FZJ-IEK3-VSA/FINE.git
+  summary: The FINE python package provides a framework for modeling, optimizing and assessing energy systems. With the provided framework, systems with multiple regions, commodities and time steps can be modeled. Target of the optimization is the minimization of the total annual cost while considering technical and enviromental constraints. Besides using the full temporal resolution, an interconnected typical period storage formulation can be applied, that reduces the complexity and computational time of the model.
```
Comment (ocefpaf) [inline recipes/fine/meta.yaml:72]: This summary is quite long. The GitHub page has:

     The FINE python package provides a framework for modeling, optimizing and assessing energy systems

File: recipes/fine/meta.yaml (hunk 2 above)
Comment (julian-belina) [reply recipes/fine/meta.yaml:3]: Thank you for your suggestions for improvements about the jinja template and the summary length. I have integrated them

**Pull request merged**